### PR TITLE
Retry on incorrect option

### PR DESCRIPTION
**Pull request by Daanikus**
It was annoying that the program would exit on a typo, especially during a large review process.

### CODE REVIEW

File: sdk/flow/confirm.go [hunk 1]
```diff
@@ -51,7 +52,8 @@ func (item *ConfirmerItem) Confirm(ctx *cli.Context) (bool, error) {
 	fmt.Scanln(&option)
 	action, err := item.action(option)
 	if err != nil {
-		return false, errors.Trace(err)
+		fmt.Fprintf(os.Stderr, "no action found for option %q", option)
```
Comment (BlakeMScurr) [inline sdk/flow/confirm.go]:
```suggestion
		util.Logger.Warnf("no action found for option %q", option)
```

From `"github.com/codelingo/lingo/app/util"`

Comment (BlakeMScurr) [inline sdk/flow/confirm.go]: We should check the error string to make sure it's due to a typo.

Comment (Daanikus) [reply sdk/flow/confirm.go]: I'm pretty sure this is the only possible error returned from that func.

Comment (Daanikus) [reply sdk/flow/confirm.go]: Will this notify the user though? Or just if debug flag is set?

Comment (BlakeMScurr) [reply sdk/flow/confirm.go]: Ah OK, yeah I missed the point here.

**Pull request merged**